styles/blocks/01-display.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 3,
+	"title": "Display",
+	"slug": "text-display",
+	"blockTypes": ["core/heading", "core/paragraph"],
+	"styles": {
+		"typography": {
+			"fontSize": "var:preset|font-size|xx-large",
```

Comment:
noting these sizes may have to be one-off sizes once we pull this into https://github.com/WordPress/twentytwentyfive/pull/422 and evaluate properly.

Reply:
for both display and subtitle